CHANGES.rst

```diff
@@ -1,6 +1,12 @@
 Changes
 =======
 
+Version 0.9.3 (UNRELEASED)
+--------------------------
+
+- Developers:
+   - Changes `git-submodule --update` command to delete from the cluster `moduels` folder the files that are not present in the shared modules directories.
```

Comment:
- Note the typo (`modules`).
- Suggestion: end the phrase with "... that are not present anymore in the shared modules source directories."